R/ss3sim_base.r

```diff
@@ -615,8 +607,19 @@ ss3sim_base <- function(iterations, scenarios, f_params,
         em_binning_params
       ))
     }
-    # Manipulate EM control file to adjust what gets estimated
 
+    # Exit this iteration and write dat file early if no EM
+    if (is.na(em_dir)) {
```

Comment:
@ChristineStawitz-NOAA this is where the code lives that saves the data sampled by {ss3sim}. I don't save just the "SS bootstraped" data. I can though if you need me to.